### PR TITLE
More providers added!

### PR DESCRIPTION
**Pull request by djap96**
**Social**
* CodePen
* Foursquare
* Dribbble
* Vimeo

**Docs**
* Python
* Unity3D
* MDN
* MDSN

### CODE REVIEW

File: providers/dribbble/dribbble.go [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+	"fmt"
+	"net/url"
+	"github.com/zquestz/s/providers"
+)
+
+func init() {
+	providers.AddProvider("dribbble", &DribbbleProvider{})
+}
+
+//Dribbble adheres to the Provider interface.
```
Comment (mbhinder) [inline providers/dribbble/dribbble.go]: Should be `DribbbleProvider`

File: providers/foursquare/foursquare.go [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+	"fmt"
+	"net/url"
+	"github.com/zquestz/s/providers"
+)
+
+func init() {
+	providers.AddProvider("foursquare", &FoursquareProvider{})
+}
+
+//Foursquare adheres to the Provider interface.
```
Comment (mbhinder) [inline providers/foursquare/foursquare.go]: FoursquareProvider

File: providers/mdn/mdn.go [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+	"fmt"
+	"net/url"
+	"github.com/zquestz/s/providers"
+)
+
+func init() {
+	providers.AddProvider("mdn", &MDNProvider{})
+}
+
+//MDN adheres to the Provider interface.
```
Comment (mbhinder) [inline providers/mdn/mdn.go]: MDNProvider

**Comment by mbhinder** [comment]
LGTM, doc strings need updating

**Comment by zquestz** [comment]
This looks great once NITs are addressed. Thanks for contributing!

**Comment by zquestz** [comment]
If you fix the nits in a separate commit, that should be fine. =)

**Comment by KeizerDev** [comment]
Doesn't he already did that? @zquestz

**Comment by zquestz** [comment]
Still see some comments that are off... for instance MDN vs MDNProvider.

File: providers/mdn/mdn.go (hunk 3 above)
Comment (KeizerDev) [inline providers/mdn/mdn.go]: @zquestz what is wrong with this?

Comment (KeizerDev) [inline providers/mdn/mdn.go]: @zquestz lol, nvm

**Comment by djap96** [comment]
Hello, I want to collaborate,
I can fix this doc errors
and put these changes in a single commit if you wan.
PD: I don't know about NIT, sorry for that

**Comment by zquestz** [comment]
NIT: change that is trivial. Nitpick =)

**Comment by zquestz** [comment]
Also feel free to add yourself to the README! This is a great set of providers =)

**Comment by djap96** [comment]
That is super cool, thanks man

**Comment by djap96** [comment]
No more nits!

**Comment by zquestz** [comment]
This looks great, no README.md addition?

File: providers/mdsn/mdsn.go [hunk 4]
```diff
@@ -0,0 +1,20 @@
+package mdsn
+
+import (
+	"fmt"
+	"net/url"
+	"github.com/zquestz/s/providers"
+)
+
+func init() {
+	providers.AddProvider("mdsn", &MDSNProvider{})
```
Comment (zquestz) [inline providers/mdsn/mdsn.go]: This should be msdn! Also wrong in the README and filename!

**Comment by djap96** [comment]
haha,that's embarrassing.
ok, give a minute

File: README.md [hunk 5]
```diff
@@ -106,6 +112,8 @@ s -b w3m cats
 * thepiratebay
 * twitchtv
 * twitter
+* unity3d
+* vimeo
 * wikipedia
```
Comment (djap96) [inline README.md]: For some git reason I needed to do this inclusions again

**Comment by zquestz** [comment]
Now that looks perfecto!

**Comment by zquestz** [comment]
Since there are a few cleanup commits, mind just squashing this to a single commit before merge?

**Comment by djap96** [comment]
Sure, I'm not git expert, so I need a sec to  figure out how to do it

**Comment by zquestz** [comment]
This might be useful: http://gitready.com/advanced/2009/02/10/squashing-commits-with-rebase.html

**Comment by djap96** [comment]
Thanks man. hope I don't messed anything with those git stuff

**Comment by zquestz** [comment]
Taking a peek now =)

**Comment by zquestz** [comment]
Looks perfect. Merging!

**Pull request merged**